03-loops-R.Rmd

```diff
@@ -214,15 +212,13 @@ length(vowels)
 > total(ex_vec)
 > ```
 
-### Processing Multiple Files
-
 We now have almost everything we need to process all our data files.
 The only thing that's missing is a function that finds files whose names match a pattern.
 We do not need to write it ourselves because R already has a function to do this called `list.files`.
 
 If we run the function without any arguments, `list.files()`, it returns every file in the current working directory.
 We can understand this result by reading the help file (`?list.files`).
-The first argument, `path`, is the path to the directory to be searched, and it has the default value of `"."` (recall from the [lesson](../shell/01-filedir.html) on the Unix Shell that `"."` is shorthand for the current working directory).
+The first argument, `path`, is the path to the directory to be searched, and it has the default value of `"."` (recall from the [lesson](../shell-novice/01-filedir.html) on the Unix Shell that `"."` is shorthand for the current working directory).
```

Comment:
Since the shell materials are now in a separate repo, the link needs to be http://swcarpentry.github.io/shell-novice/01-filedir.html